CriticalMass/Preferences.swift

```diff
@@ -7,17 +7,33 @@
 
 import Foundation
 
-class Preferences {
-    static var obersavationModeEnabled: Bool {
+class ObservationModePreferenceStore: Switchable {
+    private let defaultsKey = "observationMode"
+    private let defaults: UserDefaults
+
+    init(defaults: UserDefaults = .standard) {
+        self.defaults = defaults
+    }
+
+    func save(_ isEnabled: Bool) {
+        defaults.set(isEnabled, forKey: defaultsKey)
+        NotificationCenter.default.post(name: Notification.observationModeChanged, object: isEnabled)
+    }
+
+    func load() -> Bool? {
```

Comment:
Should `save` and `load` maybe be private? Otherwise I think naming could be improved